yb-voyager/cmd/common.go

```diff
@@ -1057,7 +1057,7 @@ type AssessmentReport struct {
 	MigrationComplexityExplanation string                                `json:"MigrationComplexityExplanation"`
 	SchemaSummary                  utils.SchemaSummary                   `json:"SchemaSummary"`
 	Sizing                         *migassessment.SizingAssessmentReport `json:"Sizing"`
-	Issues                         []AssessmentIssue                     `json:"-"` // disabled in reports till corresponding UI changes are done(json and html reports)
+	Issues                         []AssessmentIssue                     `json:"AssessmentIssues"`
```

Comment:
shall we just have the json key as `Issues` ? `Assessment` is essentially redundant, and we don't have that prefix for any other field.